tests/test.yml

```diff
@@ -1,18 +1,32 @@
 ---
 
-- hosts: localhost
+- hosts: controller
   remote_user: root
+  become: True
+  tasks:
+    - name: Installing repo for Java 8 in Ubuntu
```

Comment:
According to geerlingguy.java, if we want to install OpenJDk 8 then we need to add ppa repo for the same. Check this out: https://github.com/geerlingguy/ansible-role-java#example-playbook-install-openjdk-8